components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java

```diff
@@ -5244,6 +5360,36 @@ private ApplicationBasicInfo buildApplicationBasicInfo(ResultSet appNameResultSe
         basicInfo.setApplicationResourceId(appNameResultSet.getString(ApplicationTableColumns.UUID));
         basicInfo.setImageUrl(appNameResultSet.getString(ApplicationTableColumns.IMAGE_URL));
         basicInfo.setAccessUrl(appNameResultSet.getString(ApplicationTableColumns.ACCESS_URL));
+        // basicInfo.setInboundKey(appNameResultSet.getString(ApplicationTableColumns.INBOUND_AUTH_KEY));
```

Comment:
This line is commented. Do we need this?

Reply:
Removed the commented line.